From cc9e15280c621d455d942ef4649acaef4ede3c60 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Tue, 16 Oct 2018 11:18:37 +0500
Subject: [PATCH] geom/sysfs: fix build with glibc 2.28+

The glibc spec identifies the minor/major functions through
sysmacros.h so add the appropriate header in order to fix
the build.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 geom.c  | 1 +
 sysfs.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/geom.c b/geom.c
index 982ec55..e180cce 100644
--- a/geom.c
+++ b/geom.c
@@ -15,6 +15,7 @@
 #include <errno.h>
 #include <dirent.h>
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 #include <sys/ioctl.h>
 #include <linux/types.h>
 #include <linux/fs.h>
diff --git a/sysfs.c b/sysfs.c
index a81b148..44d602d 100644
--- a/sysfs.c
+++ b/sysfs.c
@@ -15,6 +15,7 @@
 #include <dirent.h>
 #include <limits.h>
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 #include <linux/types.h>
 
 #include "hdparm.h"
-- 
2.7.4

